nwbib/nwbib-spatial.ttl

```diff
@@ -3866,7 +3859,7 @@ nwbib-spatial:Q1541601
 
 nwbib-spatial:Q1906190
         a               skos:Concept ;
-        skos:broader    nwbib-spatial:Q512233 ;
+        skos:broader    nwbib-spatial:Q59138859 ;
```

Comment:
This is strange, as Q512233 is the ID and Q59138859 is redirected to it, see https://www.wikidata.org/entity/Q59138859.